site/profile/manifests/ts/hexrot.pp

```diff
@@ -0,0 +1,42 @@
+# @summary
+# Sets up repos and symlinks for hexrot
+
+class profile::ts::hexrot {
+  vcsrepo { '/opt/ts_config_mttcs':
+    ensure             => present,
+    provider           => git,
+    source             => 'https://github.com/lsst-ts/ts_config_mttcs.git',
+    keep_local_changes => true,
+    require            => Class[profile::core::anaconda],
```

Comment:
Why does this `vcsrepo` resource require conda?

Reply:
require removed 👍🏻